code/modules/spells/targeted/heal.dm

```diff
@@ -0,0 +1,89 @@
+/spell/targeted/heal
+	name = "Heal Other"
+	desc = "Mends basic wounds in the target."
+	abbreviation = "HL"
+	user_type = USER_TYPE_WIZARD
+	specialization = UTILITY
+	spell_levels = list(Sp_SPEED = 0, Sp_POWER = 0, Sp_RANGE = 0)
+
+	school = "transmutation"
+	charge_max = 300
+	cooldown_reduc = 75
+	cooldown_min = 150
+	invocation = "DI TIUB SEEL IM"
+	invocation_type = SpI_SHOUT
+	message = "<span class='danger'>You feel refreshed.<span>"
+	level_max = list(Sp_TOTAL = 3, Sp_SPEED = 2, Sp_POWER = 1, Sp_RANGE = 1)
+
+	max_targets = 1
+
+	amt_dam_fire = -15
+	amt_dam_brute = -15
+	amt_dam_oxy = -15
+	amt_dam_tox = -15
+
+	spell_flags = WAIT_FOR_CLICK
+
+	hud_state = "wiz_heal"
+
+/spell/targeted/heal/cast(var/list/targets, mob/user)
+	for(var/mob/living/T in targets)
+		if(spell_levels[Sp_RANGE])
+			for(var/mob/living/M in range(1, T))
```

Comment:
can you make sure this doesn't grab the original target a second time?

also, interestingly, this upgraded spell might want the ability to target a turf (imagine if you will 2 players standing with 1 tile between them, you could click that tile to heal both with AOE)

Reply:
Made sure it doesn't stack with the original target. Not sure about making it target a turf, you can already AoE heal two people by just clicking one of them.